casepro/cases/models.py

```diff
@@ -435,6 +490,7 @@ def as_json(self, full=True):
             return {
                 'id': self.pk,
                 'assignee': self.assignee.as_json(full=False),
+                'user_assignee': self.user_assignee.as_json(full=False) if self.user_assignee else None,
```

Comment:
Also need to prefetch this when searching messages since message representations include short case representations...

.. or just don't include `user_assignee` in the short form and just display the partner for simplicity when looking at messages?